.travis.yml

```diff
@@ -2,13 +2,15 @@ language: php
 matrix:
   include:
     - php: 7.1
-      dist: trusty
```

Comment:
The apt-get update is no longer working on trusty - works fine on xenial and this change does not appear to break anything.

Reply:
I have rebased to `main` and switched Travis to bionic